### ISSUE TITLE
tiny model accuracy

### ISSUE THREAD

**Issue by pawopawo** [facebookresearch/deit#32]
```
Accuracy of the network on the 50000 test images: 71.9%
Max accuracy: 71.95%
Training time 1 day, 15:01:41
```
hi, the accuracy of the tiny model I trained is 71.95, which cannot reach 72.2

**Comment by Yuxin-CV** [comment]
For me, the result of DeiT-Ti is:
```
Accuracy of the network on the 50000 test images: 72.7%
Max accuracy: 72.78%
Training time 1 day, 11:20:59
```
which is around `0.5%` higher than the result reported in the paper.

**Comment by fmassa** [comment]
Hi,

There is some small variation that can appear when running on different PyTorch / CUDA versions.
The code we released is a refactored version of the code used for the experiments of the paper, which should give the same results within noise (~0.2).

As @Yuxin-CV was able to reproduce (and even surpass) our reported results using our codebase, I'm closing this issue as normal noise from training, but let us know if you have further questions.
